android/assets/jsons/Civ V - Gods & Kings/Policies.json

```diff
@@ -18,7 +18,7 @@
 			},
 			{
 				"name": "Oligarchy",
-				"uniques": ["Units in cities cost no Maintenance", "+[50]% attacking strength for cities with garrisoned units"],
+				"uniques": ["Units in cities cost no Maintenance", "[+50]% Strength for garrisoned cities <when attacking>"],
```

Comment:
<for garrisoned cities> sounds like a conditional - I recall there's a happiness bonus for cities with garrison somewhere, and this also leads nicely into the 'ruleset-wide uniques with conditionals' that's been brewing for a while